server/request.go

```diff
@@ -265,7 +326,7 @@ func (r *RpcRequest) sendTxToRelay() {
 	if r.tx.Size() > 131072 {
 		if _, found := allowedLargeTxTargets[txTo.Hex()]; !found {
 			r.log("sendTxToRelay] large tx to not allowed target - hash: %s - target: %s", txHash, txTo)
-			r.writeRpcError("invalid target for large tx")
+			r.writeRpcError("invalid target for large tx", InternalError)
```

Comment:
It's a user error, so should probably be `InvalidRequest` rather than `InternalError`